reference_interfaces/msg/Message1kb.idl

```diff
@@ -0,0 +1,10 @@
+#include <std_msgs/msg/Header.idl>
+
+module reference_interfaces {
+  module msg {
+    struct Message1kb {
+      std_msgs::msg::Header header;
+      int64 data[1024];
```

Comment:
This message has actually the size of 8kb and not 1kb. This array size has to adjusted to 128 to fit the name - whoopsie.